pyapp/{{ cookiecutter.slug }}/bin/make_db

```diff
@@ -0,0 +1 @@
+../../../_shared/project/bin/make_db
```

Comment:
The `pyapp` and `pyramid-app` cookiecutters need symlinks to the new `make_db` and `alembic.ini` files (I didn't add these symlinks to the `pypackage` cookiecutter but we might have to if we ever have a package with DB support)